index.html

```diff
@@ -37,5 +37,200 @@ <h3>LET'S CONNECT</h3>
+              <h3>LET'S CONNECT</h3>
+              <ul>
+                  <li><a href=""><img src="image/Vector(4).png" alt=""></a></li>
+                  <li>
+                    <a href="">
+                      <img src="image/Vector(1).png" alt="">
+                      <img src="image/Vector(2).png" alt="">
+                    </a>
+                  </li>
+                  <li><a href=""><img src="image/Vector(5).png" alt=""></a></li>
+                  <li><a href=""><img src="image/Vector.png" alt=""></a></li>
+                  <li><a href=""><img src="image/Vector(3).png" alt=""></a></li>
+              </ul>    
+              <button type="button"><a href="">Get my Resume</a></button>
+            </div>
+  
+            <div class="about-side">
+              <ul>
+                <li class="list1"><p class="pa">Languages</p> <p>˅</p></li>
+                <ul class="icon-list">
+                  <li><img src="image/Ellipse1.png" alt="javaScript logo image"><p>javaScript</p></li>
+                  <li><img src="image/Ellipse1(1).png" alt="html logo image"><p>html</p></li>
+                  <li><img src="image/Ellipse1(2).png" alt="css logo image"><p>css</p></li>
+                </ul>
+                <li class="list"><p class="pa">Framework</p> <p>></p></li>
+                <li class="list"><p class="pa">Skills</p> <p>></p></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+
+
+      <!-- This is the contact form area -->
+      <div id="contact" class="contact">
+        <div class="contact-container">
+          <div class="contact-me">
+            <div>
+              <h2>Contact me</h2>
+            </div>
+          </div>
+          <div class="para">
+              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+          </div>
+          <form action="https://formspree.io/f/xyyvleev" method="post">
+              <div>
+                  <ul>
+                      <li>
+                          <label for="name"></label>
+                          <input type="text" id="name" name="user-name" placeholder="Name" required maxlength="30">
+                      </li>
+                      <li>
+                          <label for="mail"></label>
+                          <input type="mail" id="mail" name="user-mail" placeholder="name@gmail.com" required>
+                      </li>
+                      <li>
+                          <label for="msg"></label>
+                          <textarea name="msg" id="mail" name="user-message" cols="30" rows="8" placeholder="Write your message here" required maxlength="500"></textarea>
+                      </li>
+                  </ul>
+              </div>
+              <div class="btn">
+                <div>
+                  <button type="button"><a href="#">Get in Touch</a></button>
```

Comment:
Great job nicely implementing the milestone. But I see a few things that can help make your project much better. 
- The button `type` should be _submit_ and not button. This is needed to trigger the submission of the form to the provided endpoint.
- You do not need to use an `a` tag inside this button because it is currently interfering with the form submission. It takes users to the top of the page when clicked. Please remove this `a` tag.
- Your email input element should have the property `type` equal to `email` and not `mail`. Please fix this.